doc/development/guide/tests.rst

```diff
@@ -26,3 +26,58 @@ while the test coverage can be checked by running
 
 The output of the above command will show the coverage percentage of each
 file, as well as the line numbers of any lines missing test coverage.
+
+
+
+Testing Matplotlib based code
```

Comment:
This is super valuable, thank you for adding it! :fireworks: 

Could it be worth linking here even in the main code base (e.g., the docstring of the drawer)?